app/controllers/items/new.js

```diff
@@ -34,6 +47,79 @@ export default GoodcityController.extend({
   imageKeys: Ember.computed.localStorage(),
 
   i18n: Ember.inject.service(),
+
+  showAdditionalFields: Ember.computed("code", function() {
+    return !(
+      ["computer", "computer_accessory", "electrical"].indexOf(
+        this.get("code.subform")
+      ) === -1
```

Comment:
Instead of `!([...].indexOf(..) === -1)` can we use `([...].indexOf(..) > -1)` ?